File: aulas/aula04/aula04/src/pessoa.ts
class Pessoa {
  nome: string;
  idade: number;

  constructor(nome: string, idade: number) {
    this.nome = nome;
    this.idade = idade;
  }
}

const p1 = new Pessoa("Ana", 20);
const p2 = new Pessoa("Carlos", 35);

console.log(p1.nome);
console.log(p2.idade);

File: aulas/aula04/aula04/src/Pessoa.java
public class Pessoa{
    public Pessoa(String nome, int idade){
        this.nome = nome;
        this.idade = idade;
    }

    @Override
    public String toString(){
        return "Pessoa{nome='" + nome + "', idade=" + idade + "}";
    }

    public static void main(String[] args){
        Pessoa p1 = new Pessoa("Ana", 20);
        Pessoa p2 = new Pessoa("Carlos", 35);
        
        System.out.printf("")
    }
}